activation/licensed-users-count.md

```diff
@@ -22,6 +23,14 @@ The number of licensed Users within Syskit Point can be found by going to **Sett
  * **M** is the **number of licensed users in the tenant** 
  * **N** is the **number of Paid users that are included in the SysKit Point license**
 
+**The exact number and type of licenses** that are counted toward your SysKit Point subscription can be found on the **Licensed Users tile**. 
+* Click on **Users (1)** on the Home page
+* Select **Licensed Users (2)** from the drop-down menu or tile
+* **Browse (3)** by type of license assigned to users under the Licenses section
+
+![Browse Licensed Users by license](../.gitbook/assets/activation-licensed-users_browse-license.png)
```

Comment:
slici još malo smanjiti širinu. generalno je tendencija da to radimo sa svim slikama jer web page, a pogotovo docsi imaju limitiran width budući da je s lijeve strane navigacija (summary file u docsima).
tako da uvijek kad radiš screeshotove, radi ih s najmanjom mogućom širinom a da je prikazano sve što treba.
konkretno na ovoj slici možeš u column chooseru maknuti department i sing in kolonu 
i ostaviti samo Display Name, Username i Licenses kolone + smanjiš width browser windowa tako da vidiš te tri kolone pa onda napraviš screenshot